### PR TITLE
FIX an issue when re-rendering lit-html elements after rendering a non-lit-html element

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: https://github.com/storybooks/storybook/issues/5852

It seems the mountNode becomes stateful after a render has occurred in lit-html.
Since storybook is changing the dom beyond lit-html, it gets "confused" and throws an error because it's unable to remove
some nodes. I guess it's trying to do reconciliation or cleanup, and it's missing state or it's state is inconsistent
with the actual dom.

By adding a extra inner root node that gets reset when switching stories we make lit-html mount correctly

## What I did
I changed the render code of app/polymer to reset the dom (harder) when switching to a lit-html element.

### CODE REVIEW

**Comment by daKmoR** [comment]
I can confirm that this fixes https://github.com/storybooks/storybook/issues/5852

any chance this can be released? or is there anything else to do?

File: app/polymer/src/client/preview/render.js [hunk 1]
```diff
@@ -33,9 +33,13 @@ export default function renderMain({
     // Since we reuse `rootElement` for all stories, remove the stored instance first.
     // But forceRender means that it's the same story, so we want too keep the state in that case.
     if (!forceRender) {
-      render(html``, rootElement);
+      rootElement.innerHTML = '<div id="root-inner"></div>';
+    } else if (!rootElement.querySelector('[id="root-inner"]')) {
```
Comment (shilman) [inline app/polymer/src/client/preview/render.js]: shouldnt this be?
```
if(!forceReRender || !rootElement.querySelector('[id="root-inner"]')) {
 ...
}
```

**Review by shilman — CHANGES_REQUESTED** [review]

File: app/polymer/src/client/preview/render.js (hunk 1 above)
Comment (ndelangen) [reply app/polymer/src/client/preview/render.js]: You're right, i made this adjustment, good to merge?

**Review by shilman — APPROVED** [review]

**Comment by daKmoR** [comment]
woop woop thxxxx :)

**Pull request merged**